services/web/server/src/simcore_service_webserver/studies_dispatcher/handlers_redirects.py

```diff
@@ -42,6 +42,15 @@ def from_viewer(viewer: ViewerInfo) -> "ViewerQueryParams":
             viewer_version=viewer.version,
         )
 
+    @validator("file_type")
+    @classmethod
+    def ensure_extension_upper_and_dotless(cls, v):
```

Comment:
Question: Out of curiosity why was upper and not lower chosen?

Reply:
hahaha I would also go lower :D